articles/azure/azs-how-configure-powershell-users.md

```diff
@@ -49,16 +49,25 @@ Set-ExecutionPolicy RemoteSigned
 # PowerShell commands for Azure Stack Hub are installed through the PSGallery repository
 # To register the PSGallery repository, open an elevated PowerShell session and run the following command:
 Set-PSRepository -Name "PSGallery" -InstallationPolicy Trusted
-  
+
+# Install the latest version of PowerShellGet
+Install-Module -Name PowerShellGet -Force
+
 # Uninstall existing versions of Azure/Azure Stack Hub PowerShell
-Get-Module -Name Azs.*, Azure* -ListAvailable | Uninstall-Module -Force -Verbose
-  
-# Install the AzureRM.BootStrapper module. Select Yes when prompted to install NuGet
-Install-Module -Name AzureRM.BootStrapper -Verbose
+Get-Module -Name Azs.*, Azure*, Az.* -ListAvailable | Uninstall-Module -Force -Verbose
+
+# Install the Az.BootStrapper module
+# On some systems, you may need to run this command to be able to connect to PowerShell Gallery
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+Install-Module -Name Az.BootStrapper -Force -AllowPrerelease
```

Comment:
Think we need to break this up like below. Also reword the comment about the TLS command.
```
# On some older systems, you may need to explicity set the TLS 1.2 security protocol to be able to interact with PowerShell Gallery
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12

# Install the Az.BootStrapper module
Install-Module -Name Az.BootStrapper -Force -AllowPrerelease
```